app/helpers/permission_helper.rb

```diff
@@ -674,6 +674,16 @@ def can_create_custom_field_in_organization(organization)
     is_normal_user_or_admin_of_organization(organization)
   end
 
+  def can_edit_custom_field(custom_field)
+    custom_field.user == current_user ||
+      is_admin_of_organization(custom_field.organization)
+  end
```

Comment:
I checked with Erik, and he agreed we should use the same permissions as with deleting.